### PR TITLE
Add tests for ChatPageContent component

### PR DESCRIPTION
**Pull request by ruanyl**
add MessageBubble unit tests
add MessageContent unit tests
remove MessageFooter which is not needed anymore

### Description
_Describe what this change achieves._

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/tabs/chat/chat_page_content.test.tsx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+        type: 'input',
+        content: 'what indices are in my cluster?',
+        contentType: 'text',
+      },
+      {
+        type: 'output',
+        content: 'here are the indices in your cluster: .kibana',
+        contentType: 'markdown',
+        suggestedActions: [{ actionType: 'send_as_input', message: 'suggested action mock' }],
+      },
```
Comment (wanglam) [inline public/tabs/chat/chat_page_content.test.tsx:111]: Shall we add more messages section? The test case is for only display suggestions for last output. We can check if the messages displayed in the previous output message.

Comment (ruanyl) [reply public/tabs/chat/chat_page_content.test.tsx:111]: updated

File: public/tabs/chat/chat_page_content.test.tsx [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+  return {
+    MessageBubble: ({ children }: { children?: React.ReactNode }) => (
+      <div aria-label="chat message bubble">{children}</div>
+    ),
+  };
+});
+
+jest.mock('./messages/message_content', () => {
+  return { MessageContent: () => <div /> };
+});
```
Comment (joshuali925) [inline public/tabs/chat/chat_page_content.test.tsx:24]: just curious why are child components mocked?

**Review by joshuali925 — APPROVED** [review]

**Review by SuZhou-Joe — APPROVED** [review]

File: public/tabs/chat/chat_page_content.test.tsx (hunk 2 above)
Comment (ruanyl) [reply public/tabs/chat/chat_page_content.test.tsx:24]: That's a good question! There has been lots of discussions around this. For **unit test**, I tend to mock child components in RTL which are smart components, for example, the components that:
1. Access data via API call
2. Have side effects such as read/write local storage
3. Access context
4. Access global state

By mocking such components, I think it makes the tests concise and less fragile. And ofc, the child component should be covered by its own unit tests.

**Pull request merged**